content/v2.0/reference/cli/influx/_index.md

```diff
@@ -58,10 +58,4 @@ retrieving authentication tokens._
 | [user](/v2.0/reference/cli/influx/user)     | User management commands                             |
 | [write](/v2.0/reference/cli/influx/write)   | Write points to InfluxDB                             |
 
-## Flags
-| Flag            | Description                                                | Input type |
-|:----            |:-----------                                                |:----------:|
-| `-h`, `--help`  | Help for the influx command                                |            |
-| `--host`        | HTTP address of InfluxDB (default `http://localhost:9999`) | string     |
-| `--local`       | Run commands against the local filesystem                  |            |
-| `-t`, `--token` | API token to use in client calls                           | string     |
+{{% influx-cli-global-flags %}}
```

Comment:
If you remove the `help` flag from the global shortcode, use it here, but create a separate flags table with just the `help` flag.